### PR TITLE
HMAN-342  Extra NextHearingDate case types

### PR DESCRIPTION
**Pull request by mattnayler**
### JIRA link (if applicable) ###

   [HMAN-342](https://tools.hmcts.net/jira/browse/HMAN-342) _"BEFTA for NextHearingDate"_
   * [HMAN-381](https://tools.hmcts.net/jira/browse/HMAN-381) _"FTAs for CSV Processing"_ (sub-task)
   * [HMAN-382](https://tools.hmcts.net/jira/browse/HMAN-382) _"FTAs for Elastic Search processing"_ (sub-task)

### Change description ###

Extra case types to support extra functional test automation scripts in hmcts/ccd-next-hearing-date-updater#94

> Test PRs in main repos: 
> * hmcts/ccd-definition-store-api#1315
> * hmcts/ccd-data-store-api#2154

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.9"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.10_HMAN-342.1"
```
Comment (mattnayler) [inline build.gradle:9]: This should be updated to match final version number before being merged to master.

File: build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.10"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.11_HMAN-342"
```
Comment (mattnayler) [inline build.gradle:9]: This should be updated to match final version number before being merged to master.

**Review by marianadpereira — APPROVED** [review]
LGTM

File: build.gradle (hunk 2 above)
Comment (marianadpereira) [reply build.gradle]: solved in [2072449](https://github.com/hmcts/ccd-test-definitions/pull/60/commits/20724492eae906756dee1fae923d3b6fe3d20613)

**Pull request merged**